src/serverenvironment.cpp

```diff
@@ -2350,9 +2350,28 @@ bool ServerEnvironment::migrateAuthDatabase(
 		else
 			actionstream << "world.mt updated" << std::endl;
 
+		if (backend == "files") {
+			// special-case files migration:
+			// move auth.txt to auth.txt.bak if possible
+			std::string auth_txt_path =
+					game_params.world_path + DIR_DELIM + "auth.txt";
+			std::string auth_bak_path = auth_txt_path + ".bak";
+			if (!fs::PathExists(auth_bak_path))
+				if (fs::Rename(auth_txt_path, auth_bak_path))
+					actionstream << "Renamed auth.txt to auth.txt.bak"
+						     << std::endl;
+				else
+					errorstream << "Could not rename auth.txt to "
+						       "auth.txt.bak"
+						    << std::endl;
+			else
+				warningstream << "auth.txt.bak already exists, auth.txt "
+						 "not renamed" << std::endl;
+		} else errorstream << "NOT FILES?" << std::endl;
```

Comment:
Probably for debug purposes. Should be removed because another backend might come soon.